### PR TITLE
porting pblintd_cldcheck function

### PR DESCRIPTION
**Pull request by xyuan**
redo the porting of pblintd_cldcheck function

https://github.com/E3SM-Project/scream/pull/552

### CODE REVIEW

File: components/scream/src/physics/shoc/shoc_functions_f90.cpp [hunk 1]
```diff
@@ -579,6 +583,14 @@ void shoc_diag_second_moments_ubycond(SHOCSecondMomentUbycondData& d)
   d.transpose<ekat::util::TransposeDirection::f2c>();
 }
 
+void shoc_pblintd_cldcheck(SHOCPblintdCldCheckData& d)
+{
+  shoc_init(64, true);
```
Comment (jgfouca) [inline components/scream/src/physics/shoc/shoc_functions_f90.cpp:588]: Shouldn't init use nlev if it's available?

**Review by jgfouca — CHANGES_REQUESTED** [review]
init needs fix, otherwise good

File: components/scream/src/physics/shoc/shoc_functions_f90.cpp (hunk 1 above)
Comment (xyuan) [reply components/scream/src/physics/shoc/shoc_functions_f90.cpp]: yes, changes are made in the new commitment.

**Comment by xyuan** [comment]
@jgfouca Hello, this case failed at the test, however, I am unable to reproduce this error on Summit, could you please get the full output of the test and send it to my email? I was unable to read the full information on github page. thanks a lot.

**Pull request closed without merging**